hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -6,6 +6,13 @@
   build_ex:   "{build_dir}/simv"
   run_cmd:    "{job_prefix} {build_ex}"
 
+  // This option enables the following: (needed for uvm_hdl_*)
+  //  - Read capability on registers, variables, and nets
+  //  - Write (deposit) capability on registers and variables
+  //  - Force capability on registers, variables, and nets
+  // TODO Trim this flag since +f hurts performance.
+  vcs_debug_access_build_opts: "-debug_access+f"
```

Comment:
Minor nit: Would probably switch it around - `vcs_build_opt_debug_access` (we can standardize later to `{tool}_{build_opt}_{subtype}`). 

Also, `opt` instead of `opts` since it is a singular option - for plural `opts`, it will need to be a list instead of a string.

Reply:
sg, updated.